SofaKernel/modules/Sofa.Type/src/sofa/type/Mat.h

```diff
@@ -114,22 +97,22 @@ class Mat : public fixed_array<VecNoInit<C,real>, L>
 
     /// Constructor from another matrix
     template<typename real2>
-    Mat(const Mat<L,C,real2>& m)
+    constexpr Mat(const Mat<L,C,real2>& m) noexcept
     {
         std::copy(m.begin(), m.begin()+L, this->begin());
     }
 
     /// Constructor from another matrix with different size (with null default entries and ignoring outside entries)
     template<Size L2, Size C2, typename real2>
-    explicit Mat(const Mat<L2,C2,real2>& m)
+    explicit constexpr Mat(const Mat<L2,C2,real2>& m) noexcept
     {
-        Size maxL = std::min( L, L2 );
-        Size maxC = std::min( C, C2 );
+        constexpr Size maxL = std::min( L, L2 );
+        constexpr Size maxC = std::min( C, C2 );
```

Comment:
```suggestion
        constexpr Size minL = std::min( L, L2 );
        constexpr Size minC = std::min( C, C2 );
```
Funny that they are called `max` whereas they compute the min

Reply:
cannot commit your suggestion, as I have to change the subsequent loops (using those variables) :/